ui/app/components/mount-backend-form.hbs

```diff
@@ -39,12 +39,17 @@
       <FormFieldGroups @model={{@mountModel}} @renderGroup="Method Options">
         <:identityTokenKey>
           <SearchSelectWithModal
-            @id="identityTokenKey"
+            @id="key"
```

Comment:
changed this so it matched the designs for the modal. The id is used in the modal's title.